### DISCUSSION TITLE
Security of runtime license key

### DISCUSSION THREAD

**Discussion by fadebek** [dashingsoft/pyarmor#1596] [Q&A] [answered]
Hi, I had a quick question about the security of the runtime license key. Suppose I obfuscate my script using `--outer` on machine A. Can a user on machine B with a **different** license run `pyarmor gen key` and use that key to run my script? Or must the runtime key be generated with license A to work?

**Comment by jondy** [comment, marked as answer]
The key generated by `pyarmor gen key` is different from each Pyarmor license.
